requirements-nvidia.txt

```diff
@@ -3,3 +3,4 @@ dask-cuda
 jupyterlab>=3
 jupyterlab-nvdashboard
 dask_labextension
+fsspec
```

Comment:
@mloubout : we figured out this is for the dask notebook when run on GPUs . Which we don't have on CI, but @kenhester spotted it